src/pages/patch/patchRestartModal/PatchStatusCheckboxContainer.tsx

```diff
@@ -13,20 +13,44 @@ export const PatchStatusCheckboxContainer: React.FC<PatchStatusCheckboxContainer
   tasks,
   selectedTasks,
   toggleSelectedTask,
-}) => (
-  <ScrollableContainer data-cy="patch-status-selector-container">
-    {tasks.map((task) => (
-      <TaskStatusCheckbox
-        task={task}
-        selectedTasks={selectedTasks}
-        toggleSelectedTask={toggleSelectedTask}
-        key={`task_checkbox_${task.id}`}
-      />
-    ))}
-  </ScrollableContainer>
-);
+}) => {
+  const possibleListHeight = tasks.length * itemSize;
+  const listHeight =
+    possibleListHeight < maxListHeight ? possibleListHeight : maxListHeight;
+  const toggleHandler = (e) => {
+    // @ts-ignore
+    const { name } = e.target;
+    if (selectedTasks[name] !== undefined) {
+      toggleSelectedTask(name);
+    }
+  };
+  return (
+    // eslint-disable-next-line jsx-a11y/no-static-element-interactions,jsx-a11y/click-events-have-key-events
+    <div data-cy="patch-status-selector-container" onClick={toggleHandler}>
```

Comment:
Out of curiosity how does this click handler work? Is `<FixedSizeList/>` bubbling up the click event for each item in the virtualized list?

Reply:
Is there an advantage to doing it this way the docs seem to place the click handler on the actual rows https://react-window.now.sh/#/examples/list/memoized-list-items

Reply:
Children of `<FixedSizeList />`  bubble their events to the `div` wrapper. 
This change helps the performance because we are attaching only one event listener to the dom instead of many.